src/files/finding.ts

```diff
@@ -0,0 +1,67 @@
+/**
```

Comment:
can we combine this file with `./program.ts` and call the new file `resolution.ts`?